docker/quickdemo/Dockerfile

```diff
@@ -25,7 +25,7 @@
 #        .
 ARG AMORO_TAG=master-snapshot
 
-FROM eclipse-temurin:8-jdk-jammy as builder
+FROM alpine as builder
```

Comment:
There is a conflict between alpine and Flink, which causes Flink to fail to start the task manager.